core/ir/x86/decode.c

```diff
@@ -2410,6 +2436,22 @@ decode_get_tuple_type_input_size(const instr_info_t *info, decode_info_t *di)
         di->input_size = OPSZ_NA;
 }
 
+/* TODO i#6238: Not all opcodes have been reviewed.
+ * In case an opcode has not been reviewed,
+ * the default category assigned to it is DR_INSTR_CATEGORY_OTHER.
```

Comment:
I think it would be best to have it UNCATEGORIZED so it's clear it's not yet been examined.  OTHER implies someone looked at it and decided it didn't fit into any category.